deployment/sw360/docker-entrypoint.sh

```diff
@@ -113,14 +114,15 @@ fi
 if [ "$TRUSTED_CACERTS" ]; then
     for CERT_NAME in $(echo $TRUSTED_CACERTS | sed "s/,/ /g"); do
         if [ "${!CERT_NAME}" ]; then
-          echo "Trust certificate $CERT_NAME ..."
-          OUT="$(mktemp).crt"
-          echo -e "${!CERT_NAME}" > $OUT
-          keytool -keystore "$JAVA_HOME/lib/security/cacerts" \
-                  -alias "$CERT_NAME" \
-                  -storepass changeit \
-                  -noprompt \
-                  -import -file $OUT
+            echo "Trust certificate $CERT_NAME ..."
+            OUT="$(mktemp).crt"
+            echo -e "${!CERT_NAME}" > $OUT
+
+            keytool -keystore "$JAVA_HOME/lib/security/cacerts" \
+                    -alias "$CERT_NAME" \
+                    -storepass changeit \
+                    -noprompt \
+                    -import -file $OUT || echo "INFO: certificate $CERT_NAME was not imported"
```

Comment:
**Note:** This line of code is the only (significat) functional change in this PR